.builds/freebsd.yml

```diff
@@ -15,6 +15,12 @@ environment:
 tasks:
 - setup: |
     cd Nim
+    if [ "${GITHUB_PR_NUMBER+1}" ]; then
+      # this was a PR, so we need to checkout devel and try to merge into it
+      REVISION=$(git rev-parse HEAD)
+      git checkout devel
+      git merge "$REVISION"
```

Comment:
simpler?
```suggestion
      git pull origin devel
```

Reply:
Perhaps simpler, but maybe slightly less explicit? I'll try it though and see what happens.